lib/fuser_basic.h

```diff
@@ -212,6 +220,8 @@ class BasicGateFuser final : public Fuser<IO, Gate> {
           gates_lat[gate.qubits[0]].push_back(&gate);
           gates_lat[gate.qubits[1]].push_back(&gate);
           gates_seq.push_back(&gate);
+        } else {
```

Comment:
Is there any case where this `else` triggers for gates with >2 qubits?

Reply:
I think there is no such a case. The `else` in line 206 triggers for gates with >2 qubits